EEJT-000A/main/java/task2/InputValidation.java

```diff
@@ -0,0 +1,42 @@
+package task2;
+
+
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.time.format.DateTimeFormatter;
+import java.util.regex.Pattern;
+
+public class InputValidation {
+    public static boolean isNameValid(String name){
+        if(name != null
+                && Pattern.matches("[^!@#$%^&*()+= ~]+", name)){
```

Comment:
You can use String::matches method for better readability.
```suggestion
                && name.matches("[^!@#$%^&*()+= ~]+")) {
```